flow360/component/simulation/meshing_param/face_params.py

```diff
@@ -31,12 +31,13 @@ class SurfaceRefinement(Flow360BaseModel):
         description="Local maximum edge length for surface cells."
     )
     # pylint: disable=no-member
-    curvature_resolution_angle: AngleType.Positive = pd.Field(
+    curvature_resolution_angle: Optional[AngleType.Positive] = pd.Field(
```

Comment:
is there any default? Maybe we can use 15 deg?

Reply:
Added 12 deg